WoWPro_Leveling/Horde/CATA_Northern_Barrens.lua

```diff
@@ -157,7 +157,7 @@ K Sunscale Ravagers|ACTIVE|14067|S|QO|1|M|62.74,61.66|N|Kill Sunscale Ravagers.|
 K Sunscale Scytheclaws|ACTIVE|14067|S|QO|2|M|62.74,61.66|N|Kill Sunscale Scytheclaws.|
 C Waptor Twapping|QID|14068|S|U|46853|M|63.13,59.18|N|Use the Waptor Twap near Raptors, then loot the trap.|
 l Stolen Silver|ACTIVE|14067|M|63.72,58.79|QO|3|N|Loot the Stolen Silver from the Stolen Silver Lockbox.|
-C It's Gotta be the Horn|QID|865|M|64.82,57.15|N|Kill Reaperclaw and his minions. Loot his horn.|
+C It's Gotta be the Horn|QID|865|M|64.82,57.15;65.10,61.60|N|Kill Reaperclaw and his minions. Loot his horn. He wanders back and forth between the waypoints.|
```

Comment:
Adding a second coordinate to  the M tag requires using a CS, CC, or CN tag, as appropriate.
Please see the wiki, like https://github.com/Ludovicus-Maior/WoW-Pro-Guides/wiki/Addon-Syntax#waycomplete for CC.

Reply:
This just needs a CN after the coords before the N tag and its good.